src/dao/hooks/useContract.tsx

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from "react-query";
+import { DexService, MirrorNodeTokenPairResponse } from "@dex/services";
+import { DAOQueries } from "./types";
+
+type UseContactQueryKey = [DAOQueries.ContractId, string];
+
+export function useContract(address: string) {
```

Comment:
is it possible to rename it `useFetchContractId` since I guess this will be more relatable and all we are doing is converting evm to account id format.

Reply:
It is being used for both scenanrios. converting evm to account id and vice-versa. so not adding Id in the name. but added fetch in the name